packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -0,0 +1,470 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of camera;
+
+final MethodChannel _channel = const MethodChannel('plugins.flutter.io/camera');
```

Comment:
This should be calling the methdos from the interface, but I guess this is a similar case to the CameraWidget that hasn't been migrated yet?

Reply:
This channel is used by the `startImageStream` and `stopImageStream` methods. As it is the Google team is not very happy with these methods and their implementation, because they don't perform very well. So we decided in an offline meeting (between Amir, Maurice, Chris and myself) that for now we will keep these methods only available for the Android and iOS so not to break existing users until we have a better alternative. Meaning we don't add them to the platform_interface as of yet.

Reply:
Ahhh, ok, those methods would need some `assert(!kIsWeb)` or similar so people are aware that the web implementation will not handle them (at all), instead of getting a runtime error of "method channel not registered for message blah" which is more cryptic, and sounds like a bug!